lang/rust/rust_compile.mk

```diff
@@ -0,0 +1,22 @@
+define Host/Configure
+	# Required because OpenWrt Default CONFIGURE_ARGS contain extra
+	# args that cause errors
+	cd $(HOST_BUILD_DIR) && \
+	  RUST_BACKTRACE=full ./configure $(CONFIGURE_ARGS)
+endef
+
+define Host/Compile
+	cd $(HOST_BUILD_DIR) && \
+	RUST_BACKTRACE=full $(PYTHON) x.py --config ./config.toml dist cargo extended \
+	   library/std llvm-tools miri rust-dev rustc-dev
```

Comment:
Why `rust-dev` and `rustc-dev`? That's the LLVM dynamic library and libraries for using `rustc_private`; they shouldn't be needed unless you're using a custom rustc driver.